### PR TITLE
Add InfraCloud swag opportunity

### LINKED ISSUE #681: Infracloud Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Infracloud

Q. Where have you seen the announcement (URL)?  
A. : https://www.infracloud.io/blogs/infracloud-joins-hacktoberfest-2020/

Q. What kind of swag (clothing, stickers, ...)?  
A. : t-shirt, stickers

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by Archis19**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Fixes #681

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -242,6 +242,15 @@
     "dateAdded": "2020-01-02T05:30:00.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "InfraCloud",
+    "difficulty": "medium",
+    "description": "If you can get one PR merged, you will get a cool t-shirt along with some amazing stickers. Only the first 50 people will receive the swags. Jumbo Prize– A person with maximum merged PRs will also get a Kindle from our side.",
```
Comment (aslafy-z) [inline data.json:248]:
```suggestion
    "description": "Only the first 50 people will receive the swags. If you can get one PR merged to their repos, you will get a cool t-shirt along with some amazing stickers. Jumbo Prize– A person with maximum merged PRs will also get a Kindle.",
```

File: data.json [hunk 2]
```diff
@@ -242,6 +242,15 @@
...
+    "reference": "https://www.infracloud.io/blogs/infracloud-joins-hacktoberfest-2020/",
+    "image": "https://i.imgur.com/Wlfg3Xq.png",
+    "dateAdded": "2020-09-25T19:20:42.000Z",
+    "tags": ["stickers", "clothing", "kindle"]
```
Comment (aslafy-z) [inline data.json:252]:
```suggestion
    "tags": ["stickers", "clothing", "device"]
```

**Review by aslafy-z — APPROVED** [review]
LGTM after my comments are taken in account!

**Comment by aslafy-z** [comment]
Looks like InfraCloud does not like automated requests. Can you replace the url by https://twitter.com/infracloudio/status/1310980826541092865 ?

**Comment by Archis19** [comment]
I have resolved all the issues and also committed the suggestions, please check it once :)

File: data.json [hunk 3]
```diff
@@ -250,6 +260,7 @@
     "image": "https://rac4ou2q3g246tjk3rxqbt1c-wpengine.netdna-ssl.com/wp-content/uploads/2020/09/indeed_open_source_sweatshirt-1.png",
     "dateAdded": "2020-09-30T03:19:14.000Z",
     "tags": ["clothing", "hacktoberfest"]
+
```
Comment (vikaspotluri123) [inline data.json:263]:
```suggestion
```

File: data.json [hunk 4]
```diff
@@ -242,6 +242,16 @@
     "dateAdded": "2020-01-02T05:30:00.000Z",
     "tags": ["stickers"]
   },
+  {
+
```
Comment (vikaspotluri123) [inline data.json:246]:
```suggestion
```

Comment (vikaspotluri123) [reply data.json:246]: Didn't notice this the first time 😬

File: data.json [hunk 5]
```diff
@@ -242,6 +242,15 @@
...
     "tags": ["stickers"]
   },
+  {
+    "name": "InfraCloud",
+    "difficulty": "medium",
+    "description": "The first 50 people to get one PR merged into an InfraCloud repo will get a cool t-shirt, along with some amazing stickers. Jumbo Prize – The person with the most merged PRs will also get a Kindle.",
+    "reference": "https://twitter.com/infracloudio/status/1310980826541092865",
+    "image": "https://i.imgur.com/Wlfg3Xq.png",
+    "dateAdded": "2020-09-25T19:20:42.000Z",
+    "tags": ["stickers", "clothing", "device"]
```
Comment (vikaspotluri123) [inline data.json:252]:
```suggestion
    "tags": ["stickers", "clothing", "device", "hacktoberfest"]
```

**Review by vikaspotluri123 — APPROVED** [review]

**Pull request merged**